server.js

```diff
@@ -684,6 +684,33 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// Coverity Code Advisor On Demand integration
+camp.route(/^\/coverity\/ondemand\/(.+)\/(.+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var badgeType = match[1]; // streams or jobs
+  var badgeTypeId = match[2]; // streamId or jobId
```

Comment:
Coverity On Demand jobs expire after 30 days which makes including a working example that isn't stale difficult. I have just pushed a change that provides an example of a small project that can be submitted for analysis and tested against.